app/src/main/java/de/danoeh/antennapod/view/PagerIndicatorView.java

```diff
@@ -1,15 +1,17 @@
 package de.danoeh.antennapod.view;
 
+import android.animation.ArgbEvaluator;
 import android.content.Context;
 import android.content.res.TypedArray;
-import android.database.DataSetObserver;
 import android.graphics.Canvas;
 import android.graphics.Paint;
 import android.util.AttributeSet;
 import android.view.View;
+
 import androidx.annotation.Nullable;
-import androidx.vectordrawable.graphics.drawable.ArgbEvaluator;
```

Comment:
It didn't build with the change and `androidx` one is apparently declared for internal use of the package.